WinUIGallery/ControlPages/TitleBarPage.xaml.cs

```diff
@@ -209,6 +209,23 @@ private void setTxtBoxAsPasthrough(FrameworkElement txtBoxNonClientArea)
             SetClickThruRegions(rectArr);
         }
 
+        private void TitleBarHeightComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            var selectedHeight = titlebarHeight.SelectedItem.ToString();
+            var window = WindowHelper.GetWindowForElement(this);
+
+            if (selectedHeight != null && window != null && window.ExtendsContentIntoTitleBar)
+            {
+                window.AppWindow.TitleBar.PreferredHeightOption = selectedHeight switch
+                {
+                    "Tall" => TitleBarHeightOption.Tall,
+                    "Standard" => TitleBarHeightOption.Standard,
+                    "Collapsed" => TitleBarHeightOption.Collapsed,
+                    _ => TitleBarHeightOption.Tall
+                };
```

Comment:
I think it would be better to use the App.GetEnum function instead.

```suggestion
                window.AppWindow.TitleBar.PreferredHeightOption = App.GetEnum<TitleBarHeightOption>(selectedHeight);
```